AnnotatoBackend/Sources/App/DataAccess/DocumentsDataAccess.swift

```diff
@@ -30,11 +30,22 @@ struct DocumentsDataAccess {
             .map(Document.fromManagedEntity)
     }
 
-    static func update(db: Database, document: Document) -> EventLoopFuture<Document> {
+    static func update(db: Database, documentId: UUID, document: Document) -> EventLoopFuture<Document> {
         let documentEntity = DocumentEntity.fromModel(document)
 
-        return documentEntity.update(on: db)
-            .map { Document.fromManagedEntity(documentEntity) }
+        // swiftlint:disable:next first_where
+        return db.query(DocumentEntity.self)
+            .filter(\.$id == documentId)
+            .first()
+            .unwrap(or: AnnotatoError.modelNotFound(requestType: .update,
+                                                    modelType: String(describing: Document.self),
+                                                    modelId: documentId))
+            .flatMap { fetchedEntity in
+                fetchedEntity.copyPropertiesOf(otherEntity: documentEntity)
```

Comment:
Not so ideal to have to copy properties of the updated model everytime, but seems like this is the only way to replace the entire row in Fluent, since Fluent only allows updates on a fetched entity (https://docs.vapor.codes/4.0/fluent/model/#update).

An alternative is to delete the old row and create a new row using the new data but this way we lose information about `created_at`.